taworks/taform/templates/taform/ranking_status.html

```diff
@@ -12,7 +12,36 @@ <h1>Ranking Status</h1>
 <h4>Below is a listing of the status of graduate student rankings for the upcoming term's courses.</h4> 
 <p>Rankings that have been submitted can still be edited by the AC. Click on the course link to view and edit Instructor rankings.</p>
 
-<h2>***TODO Ranking status displayed here***</h2> 
+<table class="tg">
+	<tr>
+	<th class="tg-8fvv">Course</th>
+	<th class="tg-8fvv"># of Apps</th>
+	<th class="tg-8fvv">Instructor</th>
+	<th class="tg-8fvv">Instructor Email</th>
+	<th class="tg-8fvv">Ranking Status</th>
+	</tr>
+{% for c in courses %}
+<tr>
+	<td class="tg-yw4l"><a target="_blank" href="https://team4.uwaterloo.ca/taform/instructor/{{c.url_hash}}/">{{c.course_subject}}&nbsp;{{c.course_id}}</a></td>
+	<td>
+		{% for key,value in num_applicants.items %}
+			{% if forloop.parentloop.counter == forloop.counter %}
```

Comment:
Are you able to preprocess the 3 things: course, # of applicants, ranking_status
as 1 object in views.py?

So we don't need to run 2 nested for loops to render the page here and hopefully just do.

c.course_id, c.num_apps, c.rank_status in 1 shot.